### DISCUSSION TITLE
Deno is terrible. Allow just plain Node or add Go/python edge functions.

### DISCUSSION THREAD

**Discussion by DreamwareDevelopment** [supabase/supabase#31154] [Feature Requests]
Deno is terrible for many reasons. "Native TS support" means deno just ignores tsconfigs. This [issue](https://github.com/denoland/deno/issues/18474) means that it can't work with Nx which is a dealbreaker for me. I tried my best to get edge functions working in my environment but I guess I'm going to have to go with a provider that doesn't use the absolute worst JS runtime like Inngest.

**Comment by ahndwon** [comment]
Deno really sucks

**Comment by dysphasia** [comment]
The Deno requirement has stopped me from using Supabase on a number of projects. Every few months I come back to see if they've switched yet. Please consider Node, Bun, or give users multiple options for different JS Runtimes.
